staff.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
If this file should exist it should be something like /guides/getting-support and be limited in scope to how you find staff on irc. Imo

Reply:
Improving focus is always good.
I'll keep trimming.